src/main/java/com/github/manolo8/darkbot/core/manager/EffectManager.java

```diff
@@ -60,15 +60,132 @@ public enum Effect {
         UNDEFINED(-1),
         LOCATOR(1),
         PET_SPAWN(2),
+        SOLACE_NANO_CLUSTER(3),
+        DIMINISHER_WEAKEN_SHIELDS(4),
+        SPECTRUM_PRISMATIC_SHIELD(5),
+        SENTINEL_FORTRESS(6),
+        /** Cyborg uses the same effect as venom for singularity */
+        VENOM_SINGULARITY(7),
+        SABOTEUR(9),
```

Comment:
Is this just the "slowness" effect? if so it's more than just saboteour, DRC rockets can give it too. Maybe `SHIP_SLOWDOWN` and a comment saying this happens with saboteours, dcr rockets and similar?